aws-athena/ctas/proximity-dist_pin_to_airport.sql

```diff
@@ -0,0 +1,81 @@
+-- CTAS to create a table of distance to the airports for each PIN
+
+{{
+  config(
+    materialized='table',
+    partitioned_by=['year'],
+    bucketed_by=['pin10'],
+    bucket_count=1
+  )
+}}
+
+WITH distinct_pins AS (
+    SELECT DISTINCT
+        x_3435,
+        y_3435
+    FROM {{ source('spatial', 'parcel') }}
+),
+
+xy_to_airports_dist AS (
+    SELECT
+        dp.x_3435,
+        dp.y_3435,
+        ST_DISTANCE(
+            ST_POINT(dp.x_3435, dp.y_3435),
+            ST_POINT(1099852, 1935070) --ohare centroid
+        ) AS dist_to_ohare,
+        ST_DISTANCE(
+            ST_POINT(dp.x_3435, dp.y_3435),
+            ST_POINT(1142843, 1864827) --midway centroid
+        ) AS dist_to_midway        
+    FROM distinct_pins AS dp
+),
+
+--COEFFICIENTS FROM OMP-ONLY inverse square noise falloff model for O'Hare,
+-- 2019-sensor data ONLY inverse square noise falloff model for Midway:
+--ohare     0.00582564978995262
+--midway    0.00297149980129393
+
+--I_ZERO = POWER(10, -12)
+
+airport_regression AS (
+    SELECT 
+        x_3435,
+        y_3435,
+        dist_to_ohare,
+        dist_to_midway,
+        POWER(dist_to_ohare, -2) * 0.00582564978995262 AS model_output_ohare,
+        POWER(dist_to_midway, -2) * 0.00297149980129393 AS model_output_midway
+    FROM xy_to_airports_dist
+),
+
+airport_modeled_dnl AS (
+    SELECT 
+        *,
+        GREATEST(
+            0,
+            10 * LOG(10, (model_output_ohare / POWER(10,-12)))
+        ) AS dnl_ohare,
+        GREATEST(
+            0,
+            10 * LOG(10, (model_output_midway / POWER(10,-12)))
+        ) AS dnl_midway
+    FROM airport_regression
+)
+
+
+SELECT
+    pcl.pin10,
+    pcl.year, --do we want this? 
```

Comment:
**suggestion:** Move questions comments like this one to PR comments.

But yes, we do want this included in the output. Note that it's partition field in the `config` block at the top of this query. That means year is used to "chunk" the data such that a separate parquet file is saved for each year on S3. It also required year to be the last column for some reason, see the error in the workflow run that tried to generate this CTAS: https://github.com/ccao-data/data-architecture/actions/runs/6868698948/job/18679860499?pr=231

So basically, just move this column below L74.